examples/web-sdk-sample/README.md

```diff
@@ -122,4 +122,38 @@ Record data sent and received during WebSocket communication:
 
 ```
 
+### Upload files to s3 bucket
```

Comment:
add a prerequisite with link how to create an identity pool and IAM role

Reply:
fixed.